Gravity/Gravity.Test/TestClasses/GravityLevelOne.cs

```diff
@@ -34,15 +34,9 @@ public class GravityLevelOne : BaseDto
 		public RelativityFile FileField { get; set; }
 
 		[RelativityObjectField("D0770889-8A4D-436A-9647-33419B96E37E", RdoFieldType.MultipleObject, typeof(GravityLevel2))]
-		public IList<int> GravityLevel2MultipleArtifactIds { get; set; }
-
-		[RelativityMultipleObject("D0770889-8A4D-436A-9647-33419B96E37E")]
```

Comment:
These are the wrong removed attributes, no? The types on the remaining object versions should stay as is (`RelativityMultipleObject`, `RelativitySingleObject`).

Reply:
At first that is the route I took, but when I looked at it, there is no real need to the single objects and multiple object as anything other than a different field type.  I have a feeling child object will be similar.  This is much easier for the consumer.  One way to define all the fields.

Reply:
Well, child object can't be similar, because it doesn't even have a field definition.

As for single/multiple, the idea is that you don't need to declare the type because it can be inferred. Perhaps we should remove the type argument on RelativityObjectField altogether, now that we don't have the ID-list versions.

Reply:
Why another one? If you're changing the contract, you should do it in this one.

Similarly, I think this PR should remove the ObjectFieldDTOType attribute, as mentioned in a previous comment.

Reply:
Why another one?  I was trying to rush to get this done, but you are right.  I updated the readme.

As far as the remove ObjectFieldDTOType...I think that should be done.  However I would like to save that for another PR.  I will create an issue for it.